python/tvm/relay/frontend/tflite.py

```diff
@@ -792,6 +794,147 @@ def convert_not_equal(self, op):
                 'TFlite quantized NOT_EQUAL operator is not supported yet.')
         return self._convert_elemwise(_op.not_equal, op)
 
+    def convert_gather(self, op):
+        """Method to Convert TFLite GATHER operator"""
+        try:
+            from tflite.BuiltinOptions import BuiltinOptions
+            from tflite.GatherOptions import GatherOptions
+            from tflite.TensorType import TensorType
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        input_tensors = self.get_input_tensors(op)
```

Comment:
We need an assert here like `assert len(input_tensors) == 2, "input tensors length should be 2"`

Reply:
Added the assert for checking length of input tensors